_rules/html-page-lang-valid-bf051a.md

```diff
@@ -63,31 +63,31 @@ The `lang` attribute specified is neither empty ("") nor only [ASCII whitespace]
 
 #### Failed Example 1
 
-The `lang` attribute value is not a valid primary language subtag.
+This `html` element has a `lang` attribute whose value is not a [valid language subtag][].
 
 ```html
 <html lang="xyz"></html>
 ```
 
 #### Failed Example 2
 
-The `lang` attribute value has a valid primary language subtag, but a syntactically invalid region subtag.
+This `html` element has a `lang` attribute whose value is not a [valid language subtag][]. Even though the [primary language subtag][] is valid, the [region subtag] is not
```

Comment:
Missing a full stop at the end of the second sentence.

Reply:
Pro-tip: For small changes like this, you can use the "suggestion" button on github to directly make the suggested change as part of your review. That's usually faster both for the reviewer (adding the full stop is faster than writing that it is needed) and for the rule writer (github offers the possibility to accept the change in one click rather than dig through the file manually).

Reply:
Thanks ... I wasn't aware of that... and also can't see how to do that. But I'll look out for it.